pkg/kn/commands/source/apiserver/describe.go

```diff
@@ -18,23 +18,34 @@ import (
 	"errors"
 	"fmt"
 	"sort"
+	"strings"
 
 	"github.com/spf13/cobra"
+	"k8s.io/cli-runtime/pkg/genericclioptions"
 	v1alpha2 "knative.dev/eventing/pkg/apis/sources/v1alpha2"
 
 	"knative.dev/client/lib/printing"
 	"knative.dev/client/pkg/kn/commands"
 	"knative.dev/client/pkg/printers"
 )
 
+var describeExample = `
+  # Describe an ApiServer source with name 'k8sevents'
+  kn source apiserver describe k8sevents
+
+  # Describe an ApiServer source with name 'k8sevents' in YAML format
+  kn source apiserver describe k8sevents -o yaml`
+
 // NewAPIServerDescribeCommand to describe an ApiServer source object
 func NewAPIServerDescribeCommand(p *commands.KnParams) *cobra.Command {
-	apiServerDescribe := &cobra.Command{
-		Use:   "describe NAME",
-		Short: "Show details of an api-server source",
-		Example: `
-  # Describe an ApiServer source with name 'k8sevents'
-  kn source apiserver describe k8sevents`,
+
+	// For machine readable output
+	machineReadablePrintFlags := genericclioptions.NewPrintFlags("")
+
+	command := &cobra.Command{
+		Use:     "describe NAME",
+		Short:   "Show details of an api-server source",
```

Comment:
the way we word `api server` in `Short` and `Example` can be aligned
I'd go for all lower case, `api-server` lgtm. See `kn service describe -h` for example.